File: Modul 5/Soal 5/PRAK505-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h> 

void Biodata(int tahunLahir, char Namaku[], char Asal[]) {
    int tahun_sekarang = 2020; 
    int umur = tahun_sekarang - tahunLahir;

    printf("Perkenalkan Nama saya : %s \n", Namaku);
    printf("Umur Saya : %d \n", umur);
    printf("Saya Adalah Angkatan : %d \n",  tahun_sekarang);
    printf("Asal Saya Dari : %s \n", Asal);
} 

int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];
    printf("Masukkan tahun Lahir: ");
    scanf(" %d",&tahunLahir);
    printf("Masukkan Nama: ");
    scanf(" %[^\n]%*c",&Namaku);
    printf("Masukkan Asal: ");
    scanf(" %[^\n]%*c",&Asal);
    printf("\n");
    Biodata(tahunLahir, Namaku, Asal);
return 0; 
}

File: Modul 5/Soal 5/PRAK505-2410817220018-ANGGRAENI DWI ZAHRA.py
def Biodata(tahunLahir, Namaku, Asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahunLahir

    print(f"Perkenalkan Nama saya : {Namaku}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya Dari : {Asal}")

def main():
    tahunLahir = int(input("Masukkan Tahun Lahir : "))
    Namaku = input("Masukkan Nama : ")
    Asal = input("Masukkan Asal : ")
    print("")
    Biodata(tahunLahir, Namaku, Asal)

if __name__ == "__main__":
    main()